airbyte-cdk/java/airbyte-cdk/db-sources/src/main/java/io/airbyte/cdk/integrations/debezium/internals/mysql/MySqlDebeziumStateUtil.java

```diff
@@ -287,6 +288,7 @@ public JsonNode constructInitialDebeziumState(final Properties properties,
     final JsonNode asJson = serialize(offset, schemaHistory);
     LOGGER.info("Initial Debezium state constructed: {}", asJson);
 
+    assert !asJson.get(MysqlCdcStateConstants.MYSQL_DB_HISTORY).asText().isBlank();
```

Comment:
Do we really want assertions here? Shouldn't we be raising exceptions instead? I doubt that our production code runs with assertions enabled.

Reply:
Ack, will change to exception